Source/Particles/ParticleCreation/AddParticles.cpp

```diff
@@ -738,13 +738,19 @@ PhysicalParticleContainer::AddPlasmaFromFile(PlasmaInjector & plasma_injector,
 }
 
 void
-PhysicalParticleContainer::AddPlasma (PlasmaInjector const& plasma_injector, int lev, amrex::RealBox part_realbox)
+PhysicalParticleContainer::AddPlasma (PlasmaInjector& plasma_injector, int lev, amrex::RealBox part_realbox)
 {
     WARPX_PROFILE("PhysicalParticleContainer::AddPlasma()");
 
     // If no part_realbox is provided, initialize particles in the whole domain
     const Geometry& geom = Geom(lev);
-    if (!part_realbox.ok()) { part_realbox = geom.ProbDomain(); }
+    bool initial_injectcion;
```

Comment:
Could you fix the typo throughout this file?
```suggestion
    bool initial_injection;
```

Reply:
Also: could you a comment to explain what happens for `initial_injection=true` and `initial_injection=false` (my understanding is that `initial_injection=false` corresponds to the moving window).